packages/components/src/components/button/button.stories.ts

```diff
@@ -69,7 +70,62 @@ export const Primary: StoryObj = {
     disabled: false,
     'soft-disabled': false,
     variant: BUTTON_VARIANTS.PRIMARY,
+    size: PILL_BUTTON_SIZES[32],
+    color: BUTTON_COLORS.DEFAULT,
+  },
+};
+
+export const PrefixIcon: StoryObj = {
+  args: {
+    children: html`<mdc-icon name="info-circle-bold" slot="prefix-icon"></mdc-icon>
+      <span>Left Icon</span>`,
```

Comment:
does it work without a span as well?
I think we should show here how the user should do it, not what they can do here.
Ideally a consumer would just pass in the text and thats it.

Reply:
Yes, it works without span as well. Sure, I'll get them removed.